commons/src/test/java/com/ge/predix/acs/commons/web/BaseRestApiControllerAdviceTest.java

```diff
@@ -32,25 +31,16 @@
     @Test
     public void testBaseRestApiControllerAdvice() {
 
-        RestErrorHandler errorHandler = Mockito.mock(RestErrorHandler.class);
+        BaseRestApiControllerAdvice baseRestApiControllerAdvice = new BaseRestApiControllerAdvice();
```

Comment:
For the test we should use org.springframework.test.web.servlet.MockMvc to build a request and assert the return http status.  Maybe its better to move all new controller tests you added here and maintain error handling testing in one place.

Reply:
Fixed